src/mbgl/renderer/render_layer.cpp

```diff
@@ -78,21 +78,21 @@ void RenderLayer::checkRenderability(const PaintParameters& parameters,
     }
 
     if (activeBindingCount > parameters.context.maximumVertexBindingCount) {
-        Log::Info(Event::OpenGL,
```

Comment:
FWIW, I think the next message in the “else if” block could arguably be downgraded to `Info`, but the message here should always be `Error`. If you are using more than the maximum number of vertices supported by the device, that should be an error.